### DISCUSSION TITLE
Already Reblogged symbol green or purple?

### DISCUSSION THREAD

**Discussion by amaradangeli** [AprilSylph/XKit-Rewritten#758] [Lounge]
I've recently noticed both green and purple highlighting on the reblog symbol. I'm taking both to mean the same thing: that I've reblogged the post. But is there some other meaning?

**Comment by AprilSylph** [comment]
- Green = Reblogged
- Purple = Queued
- Red = Drafted

**Reply by AprilSylph** [reply]
The "remember which posts I've already reblogged" option does not differentiate, though, as it would be much more work to actually track if the post is still in your queue/drafts.
